CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+## [1.0.0] - 2022-08-24
```

Comment:
```suggestion
## [0.9.0] - 2022-08-24
```

In go a major version indicates a stable API, which we haven't reached yet (generic refactoring & co), keep it below major version 1 for now for all kiota and microsoft graph packages please. We'll bump the versions once we GA